server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java

```diff
@@ -81,7 +81,7 @@ public void update(GModelState graphicalModelState) {
 	public void subscribe(NotificationSubscriptionListener<EObject> subscriptionListener) {
 		LOGGER.debug("EcoreModelServerAccess - subscribe");
 		this.subscriptionListener = subscriptionListener;
-		this.modelServerClient.subscribe(modelUri, subscriptionListener);
+		this.modelServerClient.subscribe(modelUri, subscriptionListener, "xmi");
```

Comment:
You can use the FORMAT_XMI constant instead

Reply:
Sure, thanks for spotting!